pkg/bootstrapper/bootstrapper.go

```diff
@@ -445,9 +448,8 @@ func (wmcb *winNodeBootstrapper) initializeKubeletFiles() error {
 	return nil
 }
 
-// createKubeletService creates a new kubelet service to our specifications
-func (wmcb *winNodeBootstrapper) createKubeletService() error {
-	var err error
+// createInitialKubeletArgs defines the kubelet args required during initial kubelet start up.
+func (wmcb *winNodeBootstrapper) createInitialKubeletArgs() []string {
```

Comment:
This is more like `getInitialKubeletArgs()`.  You should update the comment stating that this function assumes `wmcb.kubeletArgs` have been populated. Ideally there should be a guard but I am not sure what that guard looks like.

Reply:
We do have some guards [here](https://github.com/openshift/windows-machine-config-bootstrapper/pull/239/files#diff-e9ddf87a450b985b61b945593604139617a74d4dbacff8b018acfe18c5880c2cR483). Show we return error if the key doesn`t exist ?

Reply:
That is not a guard. A guard would something that returns an error if the function is called before the args have been populated. 

>Should we return error if the key doesn`t exist ?

Not sure if we can do that. Some of the args are optional.